### ISSUE TITLE
PTSD 1/2 accessibility audit

### ISSUE THREAD

**Issue by pacerwow** [department-of-veterans-affairs/va.gov-team#94611]
## Issue Description
Conduct an accessibility audit of the following pages:

ptsd-intro
ptsd-type
ptsd-combat
ptsd-non-combat
walkthrough-781-choice
ptsd-intro-to-questions
ptsd-medals
ptsd-incident-date
ptsd-incident-unit-assignment
ptsd-incident-location
ptsd-inviduals-involved
ptsd-incident-support
ptsd-individuals-involved-questions
ptsd-incident-support-additional-break
ptsd-incident-description
ptsd-additional-events
ptsd-781-upload
ptsd-additional-incident
additional-remarks-781
walkthrough-781a-choice

---
## Tasks
- [ ] Conduct the accessibility audit as outlined in the Epic for the pages listed above
- [ ] Record bugs & violations in Google Sheets

## Acceptance Criteria
- [ ] An accessibility audit is performed for the pages listed above
- [ ] Results are documented in Google Sheets
- [ ] A summary is shared with our stakeholders during Sprint Review

**Comment by shannonkford** [comment]
@andaleliz Recommend closing as these are the old 0781 pages that will be deprecated.
